web-portal/pages/calendar.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div class="container info-page calendar-page">
+    <v-app>
+      <client-only>
+        <FullCalendar class="calendar" ref="fullCalendar" :options="calendarOptions" />
+      </client-only>
+    </v-app>
+  </div>
+</template>
+
+<script>
+  import { mapGetters } from 'vuex'
+  import FullCalendar from '@fullcalendar/vue'
+  import momentPlugin from '@fullcalendar/moment'
+  import dayGridPlugin from '@fullcalendar/daygrid'
+  import interactionPlugin from '@fullcalendar/interaction'
+  import timeGridPlugin from '@fullcalendar/timegrid'
+
+  export default {
+    components: {
+      FullCalendar
+    },
+    data: function () {
+      return {
+        timeInterval: null,
+        calendarOptions: {
+          plugins: [momentPlugin, dayGridPlugin, timeGridPlugin, interactionPlugin],
+          initialView: 'dayGridMonth',
+          nowIndicator: true,
+          editable: false,
+          headerToolbar: {
+            left: 'dayGridMonth,dayGridWeek,timeGridDay',
+            center: 'title',
+            right: 'prev,next'
+          },
+          titleFormat: { year: 'numeric', month: 'short', day: 'numeric' },
+          dateClick: this.handleDateClick,
+          eventClick: this.handleEventClick,
+          eventBackgroundColor: '#b7b09c',
+          eventBorderColor: '#fff',
+          eventTextColor: '#fff'
```

Comment:
Good call on using this accent color for the event backgrounds. It seems like it's only used for events that span days (but I'm guessing that's a FullCalendar behavior).

Reply:
@MatthewGidcomb Yes, indeed. I believe we can add almost any style for each element, but need to clarify the requirements to design and expected behavior for example on hover, etc.